sensor_msgs/PointCloud2 cloud
float64 x
float64 y
float64 z
float64 i




float64 x_min
float64 y_min
float64 z_min
float64 x_max
float64 y_max
float64 z_max
